asciidoc/apoc.adoc

```diff
@@ -1,23 +1,22 @@
-== Neo4j Labs Project: Awesome Procedures On Cypher (APOC)
```

Comment:
that should also work with leveloffset: -1 or ? for the adoc build?

Reply:
Maybe but I think it's cleaner to use a "standard" AsciiDoc structure with a document title and section titles. If we need to adjust the output then we should use styles but in my opinion the content should not be changed to address design/style concerns.